concepticondata/concepticon.tsv

```diff
@@ -3174,3 +3174,9 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3179	CORAL ROCK	The physical world	Limestone created from the skeleton of reef-forming stony corals (Scleractinia), displaying their growth patterns.	Person/Thing	
 3180	SHOOT (WITH SLINGSHOT)	Warfare and hunting	Use a rubber-based projectile weapon to propel projectiles against a target.	Action/Process	
 3181	MUSTARD	Food and drink	It can be a yellow sauce with a strong taste or a plant which is used to make mustard sauce.	Person/Thing	
+3182	MARRIAGE OR WEDDING	Kinship	The ceremony of marrying someone, or the state of being married	Person/Thing	
+3183	BREATHE OR BREATH	The body	The act of breathing or the air that goes into one's lungs	Other	
+3184	BE ALIVE OR LIFE	The body	The act of being alive or the life itself	Other	
+3185	DIE OR BE DEAD OR KILL	The body	To cease to live, or to be not alive, or to cause the end of a life	Other	
```

Comment:
we have the practice of ordering "or" glosses alphabetically. So "be dead or die or kill" might be more approppriate. If you add this, don't forget to add the relation to "die" proper.

Reply:
Good to know. I assumed the ordering was by word class (first verbs then nouns).

Reply:
Also, then the already existing concept of "die or be dead" needs to be renamed as well.

Reply:
We describe only ONE relation (narrower/broader) for the time being. Ths is the one we mention in our paper. The other relations were experimental and added by me for testing. But one could expand tham ad infinitum, and we had other things to do, so they are not really developed...

Reply:
sorry: this answer was meant to answer your comment above ...

Reply:
NP. Regarding the ordering, it's about 20 hyperconcepts that are not in alphabetic order and need to be renamed then.
In some few cases we might to retain the non-alphabetic order, though (e.g. he or she or it).

Reply:
Yes, there are some historical cases. I just started some time ago, when adding new concepts, to adopt for this procedure, as "foot or leg" etc. is then easier to remember, as we often have "arm or hand" vs. "leg or foot" etd. in all orders. esthetically, I prefer "die or be dead", as this is more common. we could just ignore this here. 

But I'd be inclined to ask you to add "die or be dead" as a normal concept set first, and then make the triple-one, as this is easier for future mapping as well (we have to re-map a lot of lists still, where this is not consequent).

I leave the decision of ordering to you.